src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/advice/AuthControllerAdvice.java

```diff
@@ -20,20 +20,22 @@ public boolean getIsLoggedIn(OAuth2AuthenticationToken token){
     @ModelAttribute("id")
     public String getUid(OAuth2AuthenticationToken token){
         if (token == null) return "";
-        return token.getPrincipal().getAttributes().get("id").toString();
+        return token.getPrincipal().getAttributes().get("sub").toString();
     }
 
     @ModelAttribute("login")
     public String getLogin(OAuth2AuthenticationToken token){
         if (token == null) return "";
-        return token.getPrincipal().getAttributes().get("login").toString();
+        return token.getPrincipal().getAttributes().get("name").toString();
     }
```

Comment:
There is some controversy about whether what is displayed on upper right should be name or email.  Separately from that, however, login should really be the primary email of the Google Account, not the given name.  For now this is fine, but we should probably implement a new story that allows a user to see what email they are logged in under.
@andrewhlu  could you please add that story to some epic.

Reply:
or @ryan8xia

Reply:
Got it, I can do that by simply changing "name" to "email". 
Would you like both the name and email to be stored in the model? (i.e., they could login with their email, but we could add a "name" property to the model in addition to that).

Reply:
We are suggesting doing this in a new issue.  We've already merged this PR :-)